frontend/src/user/problem/pages/index.vue

```diff
@@ -95,6 +83,16 @@ onMounted(async () => {
         {{ row.difficulty }}
       </div>
     </template>
+    <template #tags="{ row }: { row: Problem }">
```

Comment:
PaginationTable에 generic 적용되어 있어서 type assertion 없어도 row type이 `Problem`으로 뜰 거예요...!

```suggestion
    <template #tags="{ row }">
```